applications/schedview-snapshot/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+appVersion: tickets-preops-4603
```

Comment:
Our appVersions are generally numeric, consisting of the GitHub release version of the application.  We use GitHub Actions to release a new version of the package on a tag.  Note that we use numeric versions, e.g. `6.1.1` rather than `v6.1.1`.

Reply:
Confirming our exchange on slack, it's okay to use the leading v here, and not worry about making multiple tags (with and without the leading v) for each release to support the conventions of both T&S and SQuaRE?

Reply:
Yes.